crates/mem-pool/src/types.rs

```diff
@@ -49,7 +50,8 @@ impl EntryList {
         // remove lower balance withdrawals
         if let Some(withdrawal) = self.withdrawals.get(0) {
             let withdrawal_capacity: u64 = withdrawal.raw().capacity().unpack();
-            if (withdrawal_capacity as u128) > capacity {
+            let capacity = capacity.to_layer1().unwrap_or(u64::MAX);
```

Comment:
What cases go into the `.unwrap_or(u64::MAX)`?

Reply:
We can directly unwrap here. Just code style.

Reply:
I think `unwrap` directly is better.

Reply:
Ok